packages/shorebird_cli/lib/src/code_push_client_wrapper.dart

```diff
@@ -528,17 +528,17 @@ aar artifact already exists, continuing...''',
     await promotePatch(patchId: patch.id, channel: channel);
   }
 
-  Future<List<AppUsage>> getUsage() async {
+  Future<GetUsageResponse> getUsage() async {
     final progress = logger.progress('Fetching usage');
-    final List<AppUsage>? appsUsage;
+    final GetUsageResponse? usage;
     try {
-      appsUsage = await codePushClient.getUsage();
+      usage = await codePushClient.getUsage();
```

Comment:
You can declare `usage` in the scope of this try and return below the `progress.complete()`